tests/operators/test_collapse.py

```diff
@@ -347,12 +327,12 @@ def test_collapse_by_proportion_units(ensemble_cube):
 
 def test_collapse_by_proportion_name(ensemble_cube):
     """Test renaming of cube when collapsing by proportion method."""
-    expected_name = f"probability_of_{ensemble_cube.name()}_eq_276"
+    name = ensemble_cube.long_name if ensemble_cube.long_name else ensemble_cube.name()
```

Comment:
```suggestion
    name = ensemble_cube.long_name
```
No need to add this logic in our test setup.